sylvia/tests/querier.rs

```diff
@@ -0,0 +1,125 @@
+use cosmwasm_std::{Addr, Response, StdError, StdResult};
+use cw_storage_plus::Item;
+use serde::{Deserialize, Serialize};
+use sylvia::types::{ExecCtx, InstantiateCtx, QueryCtx};
+use sylvia::{contract, interface};
+
+#[derive(
+    Serialize, Deserialize, Clone, PartialEq, Eq, sylvia::schemars::JsonSchema, Debug, Default,
+)]
+pub struct CountResponse {
+    pub count: u64,
+}
+
+pub mod counter {
+    use super::*;
```

Comment:
I already roust you for that on the other review. Don't. Show explicitly what you import - this is not an additional layer of test abstraction; it is a standalone module implementing trait for the contract, which could as well be on a separate file. I don't see the structure of generated code this way.